src/types/jukebox.ts

```diff
@@ -0,0 +1,16 @@
+import firebase from "firebase/app";
+
+export type JukeboxMessage = {
+  from: string;
+  text: string;
+  tableId: string;
+  ts_utc: firebase.firestore.Timestamp;
```

Comment:
have you found a way to parse the firestore timestamp? I couldn't do that. 🙈 

if you know, please, share how can we do that.
If you don't, then let's stick to `utcMilliseconds` -> `Date.now()` to be able to display the date/time when needed.

Reply:
you can take seconds and convert it to miliseconds and then create new Date from it 
![image](https://user-images.githubusercontent.com/56254806/131972537-d2d67d10-cd54-4642-ad05-7e146eb218d4.png)

Reply:
not super convenient.. yeah, i saw that.